src/commands/createComponent.php

```diff
@@ -1,18 +0,0 @@
-<?php declare(strict_types=1);
```

Comment:
Why remove this command? I think it might be useful sometimes if you want to add a component explicitly. Do you think it's a "footgun"?

Reply:
Was this command not exposed through run-command and was there no test for it?